FreeRTOS/FreeRTOS-Plus-TCP
# Remove unnecessary #ifndef

Description
-----------

When using lint (or Coverity) the configASSERT macro should not be defined. And therefore, the #ifndef is unnecessary.

Test Steps
-----------

Not-applicable

Related Issue
-----------

Not-applicable

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
